dojo/tools/mend/parser.py

```diff
@@ -188,7 +192,8 @@ def _build_common_output(node, lib_name=None):
                 dynamic_finding=True,
                 cvssv3=cvss3_vector,
                 cvssv3_score=float(cvss3_score) if cvss3_score is not None else None,
-                impact=impact,
+                impact=impact if impact is not None else None,
+                steps_to_reproduce="**Locations Found**: " + ", ".join(locations) if locations is not None else None,
```

Comment:
I would suggest separating these paths with newlines (`\n`). It can be harder to tell where one path ends and the next begins when they're all strung together with commas, especially if there are spaces/commas in those paths.

Reply:
Newline paths break table formatting - so I avoid those where possible.  If the goal is to send these to Jira, a newline separator will break any grouping in a table.  I encountered this with the ASFF parser modifications I made, so I'd like to continue to use a comma instead of a newline.